lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserSwift.cpp

```diff
@@ -230,13 +191,11 @@ lldb::TypeSP DWARFASTParserSwift::ParseTypeFromDWARF(const SymbolContext &sc,
     type_sp = TypeSP(new Type(
         die.GetID(), die.GetDWARF(),
         preferred_name ? preferred_name : compiler_type.GetTypeName(),
-        is_clang_type ? dwarf_byte_size
-                      : compiler_type.GetByteSize(nullptr),
+        // FIXME: We don't have an exe_scope here by design.
```

Comment:
is this really a FIXME or just a comment?

Reply:
Updated.